### PR TITLE
Address input

### PR DESCRIPTION
**Pull request by abrande**
- Added Country and State selection
- Added Checkbox for RBU Customer selection
- Renamed AccountRole to Product and related translations
- Style changes

I did **not** add nearly 5k lines of code. Those are the snapshots automatically created by jest

### CODE REVIEW

File: app/components/SignUp/AddressSelectors/StateSelect.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  render() {
+    const { input, country, label } = this.props;
+    return (
+      <div className="InputField">
+        <hx-select-control>
+          <RegionDropdown
+            {...input}
+            name={input.name}
+            className="form-control"
```
Comment (nicko-winner) [inline app/components/SignUp/AddressSelectors/StateSelect.js]: I remember from form-control bootstrap, does helix use it? the usually prefix their class names with hx...Does this class have a definition in CSS?

File: app/components/SignUp/AddressSelectors/CountrySelect.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <div className="InputField">
+        <hx-select-control>
+          <CountryDropdown
+            {...input}
+            name={input.name}
+            className="form-control"
+            value={country}
+            valueType="short"
+            id={input.name}
+            onChange={(val) => input.onChange(onCountryChange(val))}
```
Comment (explorigin) [inline app/components/SignUp/AddressSelectors/CountrySelect.js]: I see why you had to do this but does all this redirection make it hard to read. However, you can't be sure that `onCountryChange` will return any meaningful value. To be more robust, you'll need to call `onCountryChange` and `input.onChange` separately. Ideally `onCountryChange` would react to input's `onChange` event so we don't have so much redirection going on.

File: app/components/SignUp/AddressSelectors/StateSelect.js (hunk 1 above)
Comment (abrande) [reply app/components/SignUp/AddressSelectors/StateSelect.js]: Ah, This should have been deleted. Doing this now

File: app/components/SignUp/AddressSelectors/CountrySelect.js (hunk 2 above)
Comment (abrande) [reply app/components/SignUp/AddressSelectors/CountrySelect.js]: I was having an issue with it storing the country in the form state without having this redirection. However what I had is magically working now. It looks like this 
```
            <Field
              ...
              country={this.state.country}
              onCountryChange={this.onCountryChange}
            />
```

and changing the CountryDropdown to 

```
          <CountryDropdown
            ...
            value={country}
            onChange={onCountryChange}
          />
```

**Pull request merged**